Bruin Bite/Models/ErrorLogger.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  ErrorLogger.swift
+//  Bruin Bite
+//
+//  Created by Kameron Carr on 9/1/18.
+//  Copyright © 2018 Dont Eat Alone. All rights reserved.
+//
+
+import Foundation
+
+enum ErrorType: String {
+    case error = "❤️ -ERROR- ❤️"
+    case warning = "💛 -WARNING- 💛"
+    case info = "💚 -INFO- 💚"
+    case debug = "💙 -DEBUG- 💙"
+    case verbose = "💜 -VERBOSE- 💜"
+    case other = "🖤 -OTHER- 🖤"
+    case none = ""
```

Comment:
While it's tempting to use `rawValue`s to determine the header for the different types here, that's not the intended use case for `rawValue`. I would instead add a function in the enum that returns the desired header string based on its own type and leave the raw values blank for now.

so the function would be added inside the enum and would look something like this:
```swift
func header() -> String {
    switch self {
        case .error:
            return "❤️ -ERROR- ❤️"
        // Handle all the other cases here (except maybe none, because that will be the same as default
        default:
            return ""
    }
}
```